corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -106,7 +107,13 @@ def run(self):
         pillow_logging.info("Starting pillow %s" % self.__class__)
         with configure_scope() as scope:
             scope.set_tag("pillow_name", self.get_name())
-        self.process_changes(since=self.get_last_checkpoint_sequence(), forever=True)
+        if self.process_num == 0  and self.is_dedicated_migration_process:
```

Comment:
I think this is redundant since it's always set to `is_dedicated_migration_process=dedicated_migration_process and (process_num == 0)`, right? I think it'd be less confusing if this line was just

```suggestion
        if self.is_dedicated_migration_process:
```
unless I'm missing something

Reply:
No, Danny, you're correct. In order to capture both process_num and dedicated_migration_process, I placed it in that way. We can remove it